src/main/webapp/availability.html

```diff
@@ -44,505 +44,14 @@ <h5 class="text-center lead">Update your availability to be an Interviewer for t
-              <option>00</option>
-              <option>15</option>
-              <option>30</option>
-              <option>45</option>
-            </select>
-          </div>
-          <div class="col">
-            <select class="form-control">
-              <option>AM</option>
-              <option>PM</option>
-            </select>
-          </div>
-        </div>
-      </form>
-      <br>
-      
-      <h5 class="font-weight-bold">Saturday 7/4</h5>
-      <form>
-        <div class="form-row">
-          <p class="form-labels">Start Time:</p>
-          <div class="col">
-            <select class="form-control">
-              <option>1</option>
-              <option>2</option>
-              <option>3</option>
-              <option>4</option>
-              <option>5</option>
-              <option>6</option>
-              <option>7</option>
-              <option>8</option>
-              <option>9</option>
-              <option>10</option>
-              <option>11</option>
-              <option>12</option>
-            </select>
-          </div>
-          <p class="form-labels">:</p>
-          <div class="col">
-            <select class="form-control">
-              <option>00</option>
-              <option>15</option>
-              <option>30</option>
-              <option>45</option>
-            </select>
-          </div>
-          <div class="col">
-            <select class="form-control">
-              <option>AM</option>
-              <option>PM</option>
-            </select>
-          </div>
-          <p class="form-labels end-time">End Time:</p>
-          <div class="col">
-            <select class="form-control">
-              <option>1</option>
-              <option>2</option>
-              <option>3</option>
-              <option>4</option>
-              <option>5</option>
-              <option>6</option>
-              <option>7</option>
-              <option>8</option>
-              <option>9</option>
-              <option>10</option>
-              <option>11</option>
-              <option>12</option>
-            </select>
-          </div>
-          <p class="form-labels">:</p>
-          <div class="col">
-            <select class="form-control">
-              <option>00</option>
-              <option>15</option>
-              <option>30</option>
-              <option>45</option>
-            </select>
-          </div>
-          <div class="col">
-            <select class="form-control">
-              <option>AM</option>
-              <option>PM</option>
-            </select>
-          </div>
-        </div>
-      </form>
-      <br>
-
-      <button type="submit" class="btn btn-primary mb-2 submit-button">Update Availability</button>
+      <!-- This table-container is filled with a JSP page onload.-->
+      <div id="table-container">
+        <!-- This is a place holder for when the page is slow to load.-->
+        <p class="text-center">Time Table</p>
```

Comment:
Nit: "Time table" is not something a user will understand.

Maybe "Loading interview availability..."?

Reply:
Done